Classes/Charts/Models/ChartViewModel.swift

```diff
@@ -0,0 +1,29 @@
+// Copyright 2025 Pera Wallet, LDA
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//    http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//   ChartViewModel.swift
+
+struct ChartViewModel: Hashable, Equatable {
+    let period: ChartDataPeriod
+    let chartValues: [ChartDataPoint]
```

Comment:
Can you make `ChartDataPeriod` and `ChartDataPoint` hashable and equatable? When you do that then `ChartViewModel` will also be compatible with those protocols.